### DISCUSSION TITLE
How to add custom field in a page?

### DISCUSSION THREAD

**Discussion by maxZBS** [vercel/next.js#49838] [App Router]
Hello! In previous version nextjs when I used pages directory. I could create custom field in any page for check authorization with auth provider. Like this MyOrdersPage.isOnlyUser = true. How to use it in current version with app directory?

**Comment by devjiwonchoi** [comment]
You could probably call it from the layout.js and let it work as an auth provider. Not sure if this is what you are looking for.
ref: https://nextjs.org/docs/app/building-your-application/routing/pages-and-layouts
